cleantalk.php

```diff
@@ -2190,7 +2192,7 @@ function apbct_store__urls()
     global $apbct;
 
     if (
-        $apbct->data['cookies_type'] === 'none' || // Do not set cookies if option is disabled (for Varnish cache).
+        //$apbct->data['cookies_type'] === 'none' || // Do not set cookies if option is disabled (for Varnish cache).
```

Comment:
Do not afraid to remove unused code, not just comment it.

Reply:
Done